expr/parse.go

```diff
@@ -112,12 +112,13 @@ func Parse(e string) (*expr, string, error) {
 		return parseConst(e)
 	}
 
-	if strings.HasPrefix(e, "True") || strings.HasPrefix(e, "true") {
-		return &expr{bool: true, str: e[:4], etype: etBool}, e[4:], nil
-	}
-
-	if strings.HasPrefix(e, "False") || strings.HasPrefix(e, "false") {
-		return &expr{bool: false, str: e[:5], etype: etBool}, e[5:], nil
+	if val, wasBool := strToBool(e); wasBool {
+		// 'false' is 5 chars, 'true' is 4
+		size := 5
+		if val {
+			size = 4
+		}
+		return &expr{bool: val, str: e[:size], etype: etBool}, e[size:], nil
```

Comment:
why is etype etBool here and not etString?

Reply:
This was an unquoted boolean (e.g. `false` not `'false'` or `"false"`). This makes it unambiguously an etBool

Reply:
ah you got me. You know this code better now than I do :smile: